tests/test_health.py

```diff
@@ -33,12 +33,14 @@ def test_default_health():
     server.listen()
 
     client = TChannel("health_test_client")
-    service = thrift_request_builder(
-        service='meta',
-        thrift_module=Meta,
+
+    service = thrift.load(
+        'health_test_server',
```

Comment:
This one makes sense because it's a downstream call and this is the name hyperbahn would get.